### PR TITLE
issue #85: fix JitPack build

### PR DESCRIPTION
**Pull request by siordache**

### CODE REVIEW

File: jitpack.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+jdk:
+  - openjdk11
+before_install:
+  - ./prepareEnvironment.sh
+install:
+  - mvn clean install -DskipTests
```
Comment (gunnarmorling) [inline jitpack.yml]: Thanks! Why skipping the tests, wouldn't make sense to run them to make sure the "released" artifact is sound?

File: jitpack.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+jdk:
+  - openjdk11
+before_install:
+  - ./prepareEnvironment.sh
```
Comment (gunnarmorling) [inline jitpack.yml]: What is this one doing?

Comment (siordache) [reply jitpack.yml]: I used the [custom commands example](https://jitpack.io/docs/BUILDING/#custom-commands) as a template and assumed that `prepareEnvironment.sh` is required. But, in reality, only the `jdk` needs to be specified. I removed now the unnecessary lines.

**Comment by gunnarmorling** [comment]
Nice one, thanks, @siordache!

**Pull request merged**